applications/rovioli/src/rovio-localization-handler.cc

```diff
@@ -357,10 +440,9 @@ bool RovioLocalizationHandler::getLocalizationReprojectionErrors(
   }
 
   CHECK_EQ(lc_reprojection_errors->size(), filter_reprojection_errors->size());
-
-  constexpr double kMinSuccessfulReprojectionRatio = 0.5;
-  return (static_cast<double>(lc_reprojection_errors->size()) /
-          num_matches_processed) > kMinSuccessfulReprojectionRatio;
+  CHECK_GT(num_matches_processed, 0);
+  return static_cast<double>(lc_reprojection_errors->size()) /
```

Comment:
Shouldn't we also check the absolute value?

Reply:
We could add an additional check here as well, but wouldn't the localizer fail if the number of matches is too low and there wouldn't be a result here to process anyways?